frontend/rust-lib/flowy-core/src/lib.rs

```diff
@@ -323,13 +320,3 @@ impl ServerUser for ServerUserImpl {
     self.upgrade_user()?.workspace_id()
   }
 }
-
-pub struct MutexAppFlowyCore(pub Rc<Mutex<AppFlowyCore>>);
```

Comment:
the MutexAppFlowyCore is used in tauri project. it's wrapped in State which requires send + sync

Reply:
i reverted it. Feel free to modify